src/cdk/testing/tests/webdriver-test.bzl

```diff
@@ -0,0 +1,34 @@
+load("//tools:defaults.bzl", "jasmine_node_test")
+load("@io_bazel_rules_webtesting//web:web.bzl", "web_test")
+load("//tools/server-test:index.bzl", "server_test")
+
+def webdriver_test(name, tags = [], **kwargs):
+    jasmine_node_test(
+        name = "%s_jasmine_test" % name,
+        data = [
+            "@npm//:node_modules",
```

Comment:
Bringing in the whole `node_modules` is usually causing extremely large runfile tree's and slower builds. Can we just use fine-grained deps?

Reply:
good catch, that's just something I did when I was trying to get things working